app/src/main/java/com/google/android/samples/socialite/ui/camera/CameraViewModel.kt

```diff
@@ -93,12 +98,42 @@ class CameraViewModel @Inject constructor(
         .setQualitySelector(QualitySelector.from(Quality.HIGHEST))
         .build()
 
-    private val videoCaptureUseCase = VideoCapture.Builder(recorder)
-        .build()
-
     private var currentRecording: Recording? = null
     private lateinit var recordingState: VideoRecordEvent
 
+    init {
+        val videoCaptureBuilder = VideoCapture.Builder(recorder)
+
```

Comment:
nit: any reason for the space here?

Reply:
i think just habit for personal preference for code readability, but removed